### PR TITLE
add comment element

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

File: src/components/molecules/Comment/Comment.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                </Text>
+            </div>
+        </div>
+    );
+};
+
+Comment.propTypes = {
+    commentText: PropTypes.string.isRequired,
+    username: PropTypes.string.isRequired,
+    date: PropTypes.string.isRequired
```
Comment (Veryyapeee) [inline src/components/molecules/Comment/Comment.jsx:29]: that should be a date element, and we should convert that to proper string directly in the component,
for example here we will receive the UTC date format, and here we want to convert that for like 'HH:MM' format or smth

File: src/components/molecules/Comment/Comment.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
```
Comment (Veryyapeee) [inline src/components/molecules/Comment/Comment.jsx:1]: Tested this time? :D

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/molecules/Comment/Comment.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                <Text type='primary' size='medium'>
+                    {commentText}
+                </Text>
+            </div>
+        </div>
+    );
+};
+
+Comment.propTypes = {
+    commentText: PropTypes.string.isRequired,
```
Comment (Veryyapeee) [inline src/components/molecules/Comment/Comment.jsx:27]: also this prop should be just `children`

File: src/components/molecules/Comment/Comment.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        gap: 2rem;
+        align-items: center;
+    }
+
+    &__text {
+        margin: 1rem 3rem;
+        padding: 1rem 1rem;
+        background-color: $comment-backgroundColor;
+        border-radius: 10px;
+        box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```
Comment (Veryyapeee) [inline src/components/molecules/Comment/Comment.scss:18]: variabless.scss :P

**Comment by Veryyapeee** [comment]
I think that font size is too big on mobile

File: src/components/molecules/Comment/Comment.scss [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    }
+}
+
+@include for-phone-only {
```
Comment (Veryyapeee) [inline src/components/molecules/Comment/Comment.scss:22]: On Main resposinve font change is handled, pull Main and you won't probably need these styles

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/atoms/Date/Date.jsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import * as dayjs from 'dayjs';
+
+const DEFAULT_FORMAT = 'DD MM YYYY hh:mm';
+
+export const Date = ({ format = DEFAULT_FORMAT, children }) => {
+    const formatedDate = dayjs(children).format(format);
```
Comment (Veryyapeee) [inline src/components/atoms/Date/Date.jsx:7]: Such a overkill, we should just create helper function to parse date which will accept the two parameters
date, format, with default format
and return formatted date, we really don't need the separate component for that, overkilllll

File: src/components/molecules/Comment/Comment.scss (hunk 5 above)
Comment (Veryyapeee) [reply src/components/molecules/Comment/Comment.scss:22]: ?? @Mariusz-web

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

**Review by T0ffik — APPROVED** [review]

File: src/components/molecules/Comment/Comment.jsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+export const Comment = ({ username, date, children }) => {
+    return (
+        <div className='comment'>
+            <div className='comment__info'>
+                <Text
+                    type='primary'
+                    size='small'
+                    customClass='comment__info__date'
+                >
+                    <Text>{dateFormat(date)}</Text>
```
Comment (Veryyapeee) [inline src/components/molecules/Comment/Comment.jsx:16]: You don't need text inside text, what for?

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/utils/format.js [hunk 8]
```diff
@@ -0,0 +1,5 @@
+import * as dayjs from 'dayjs';
+
+export function dateFormat(date, format = 'DD MM YYYY hh:mm') {
```
Comment (Veryyapeee) [inline src/utils/format.js:3]: Use arrow function

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**